netlify.toml

```
@@ -40,3 +40,15 @@
   force = true
 
   headers = {Apollo-Proxy-Rule = "from-root-config"}
+
+[[redirects]]
+  from = "/legacy-cli/:platform/:version"
```

Doesn't seem like we're using `:platform` or `:version`, so might be better to just leave off the capture groups if they're not being used on the `to` end.

```suggestion
  from = "/legacy-cli/*"
```